target/linux/mvebu/image/cortexa53.mk

```diff
@@ -32,6 +32,17 @@ define Device/globalscale_espressobin-emmc
 endef
 TARGET_DEVICES += globalscale_espressobin-emmc
 
+define Device/globalscale_espressobin-ultra
+  $(call Device/Default-arm64)
+  DEVICE_VENDOR := Marvell
```

Comment:
I don't understand why vendor is Marvell for this variable and globalscale for the image, but that is consistently inconsistent here, so probably we shouldn't start a new battle here.

Reply:
Doesn't make much sense to me either to be honest, but I was just following convention.